src/Servers/HttpSys/src/ServerDelegationPropertyFeature.cs

```diff
@@ -10,14 +10,9 @@ internal class ServerDelegationPropertyFeature : IServerDelegationFeature
     private readonly ILogger _logger;
     private readonly UrlGroup _urlGroup;
 
-    public ServerDelegationPropertyFeature(RequestQueue queue, ILogger logger)
+    public ServerDelegationPropertyFeature(UrlGroup urlGroup!!, ILogger logger)
```

Comment:
Don't use `!!`, it was dropped. You can use `??`.
`_urlGroup = urlGroup ?? throw new ArgumentException(nameof(urlGroup));`

Reply:
It was :( VS suggested using so I went with it will update.